script.js

```diff
@@ -0,0 +1,49 @@
+function addZeroes(num) {
+    const stringNum = num.toString().split('.')[1]
+    const len = stringNum && stringNum.length > 2 ? stringNum.length : 2
+    return Number(num).toFixed(len)
+}
+
+$('#target').submit(function(e){
+    e.preventDefault()
+    const bill = $('.money').val()
+    const tip = $('.radio:checked').val()
+    const tipPercentage = (tip / 100) + 1
+    const totalAmountUnrounded = bill * tipPercentage
+        const totalAmountRounded = Math.round(totalAmountUnrounded * 100) / 100
+    const tipAmountUnrounded = totalAmountRounded - bill
+        const tipAmountRounded = Math.round(tipAmountUnrounded * 100) / 100
+
+        let finalTip = addZeroes(tipAmountRounded)
+        let finalBill = addZeroes(totalAmountRounded)
+
```

Comment:
I actually really like this little indentation trick. At first I thought it was messy, but now I realize what you were trying to do. 

A linter might scream at you, but this is one of those cases where human judgment could be better than automated style checking :-) 

In this case, applying the [Principle of Least Surprise](https://wiki.c2.com/?PrincipleOfLeastAstonishment) to language conventions, I would do the following to not throw traditional javascript developers off: (including comments):

```suggestion
    // get basic values
    const bill = $('.money').val()
    const tip = $('.radio:checked').val()
    
    // calculate the tip
    const tipPercentage = (tip / 100) + 1
    
    // calculate the total amount and round it
    const totalAmountUnrounded = bill * tipPercentage
    const totalAmountRounded = Math.round(totalAmountUnrounded * 100) / 100
    
    // calculate the tip amount and round it
    const tipAmountUnrounded = totalAmountRounded - bill
    const tipAmountRounded = Math.round(tipAmountUnrounded * 100) / 100

    // add zeroes to get final tip and bill amounts
    let finalTip = addZeroes(tipAmountRounded)
    let finalBill = addZeroes(totalAmountRounded)

```